cmd/ctr/commands/resolver.go

```diff
@@ -94,6 +99,7 @@ func GetResolver(ctx gocontext.Context, clicontext *cli.Context) (remotes.Resolv
 		},
 		ExpectContinueTimeout: 5 * time.Second,
 	}
+	loadCertsDir(tr.TLSClientConfig, clicontext.String("certs-dir"))
```

Comment:
Shouldn't this be an Opt and not specific to `ctr`?

Reply:
I had considered adding `func WithDockerTLSCerts(string) AuthorizerOpt`, but that would require [`NewDockerAuthorizer(opts ...AuthorizerOpt) Authorizer`](https://godoc.org/github.com/containerd/containerd/remotes/docker#NewDockerAuthorizer) to return an error.

Also, the implementation is likely to be non-robust when `http.Client.Transport` doesn't support TLS.